presto-spi/src/main/java/com/facebook/presto/spi/block/ByteArrayBlockBuilder.java

```diff
@@ -29,13 +29,15 @@
         implements BlockBuilder
 {
     private static final int INSTANCE_SIZE = ClassLayout.parseClass(ByteArrayBlockBuilder.class).instanceSize();
+    private static final Block NULL_VALUE_BLOCK = new ByteArrayBlock(1, new boolean[]{true}, new byte[1]);
```

Comment:
fix fmt

Reply:
What has to be fixed here ?

Reply:
see https://github.com/airlift/codestyle (call it nitpicking, but there should be a space between `[]` and `{true}`.)

as unimportant as it might sound, having to exclude undesired changes when composing a commit might annoy and this will happen when a next person edits the file

Reply:
Thanks for pointing it out @findepi. I will change them